### PR TITLE
ENH: Update CI to the remote module GitHub Action

### PR DESCRIPTION
**Pull request by thewtex**
Common configuration is maintained in the GitHub Action repository.

### CODE REVIEW

**Review by jhlegarreta — APPROVED** [review]
Thanks Matt :100:.

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -3,279 +3,14 @@ name: Build, test, package
...
-        password: ${{ secrets.pypi_password }}
+  cxx-build-workflow:
+    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@eebccc8d50a25fac571324ae56dd254bd64607a8
+    with:
+      itk-cmake-options: '-DITK_BUILD_DEFAULT_MODULES:BOOL=OFF -DITKGroup_IO:BOOL=ON'
+
+  python-build-workflow:
+    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@eebccc8d50a25fac571324ae56dd254bd64607a8
+    with:
+      test-notebooks: true
```
Comment (dzenanz) [inline .github/workflows/build-test-package.yml]: This should be false when there are no notebooks.

**Review by dzenanz — COMMENTED** [review]
Let's see the CI test run.

File: .github/workflows/build-test-package.yml (hunk 1 above)
Comment (dzenanz) [reply .github/workflows/build-test-package.yml]: `test-linux-notebooks` action fails: `ERROR: file or directory not found: examples/*.ipynb`.

**Review by dzenanz — APPROVED** [review]

File: .github/workflows/build-test-package.yml (hunk 1 above)
Comment (jhlegarreta) [reply .github/workflows/build-test-package.yml]: Done in aec9dc7.

**Pull request merged**